packages/node_modules/@ciscospark/react-component-utils/src/badge.test.js

```diff
@@ -0,0 +1,337 @@
+import {getBadgeState} from './badge';
+import * as NotificationsConstants from './constants';
+
+describe('Get badge state', () => {
+  it('returns NOTIFICATIONS_BADGE_UNREAD for a unread 1on1 convo', () => {
+    const convo = {isOneOnOne: true};
+    expect(getBadgeState(convo, true, false)).toEqual(NotificationsConstants.NOTIFICATIONS_BADGE_UNREAD);
+  });
+
+  it('returns NOTIFICATIONS_BADGE_NONE for a read 1on1 convo', () => {
+    const convo = {isOneOnOne: true};
+    expect(getBadgeState(convo, false, false)).toEqual(NotificationsConstants.NOTIFICATIONS_BADGE_NONE);
+  });
+
+  it('returns UNREAD for an unread group convo (golobal ALL)', () => {
+    const convo = {
```

Comment:
Nice to have: you could do a `beforeEach` and set the convo object to a default state and just do one or two lines in each test block to change the state. Would reduce the large code blocks in each block.